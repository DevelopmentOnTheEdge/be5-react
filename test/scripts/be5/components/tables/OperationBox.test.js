import React from 'react';
import renderer from 'react-test-renderer';
import '../../../../../src/scripts/be5/be5init.js'
import Chart from "../../../../../src/scripts/be5/components/charts/Chart";
import testData from "../../testData.json";
import {OperationBox} from "../../../../../src/scripts/be5";

test('snapshot', () => {
    const component = renderer.create(
            <OperationBox operations={testData.orderedOperations.included[0]} frontendParams={{documentName: 'test'}}/>

    );

    expect(component.toJSON()).toMatchSnapshot();
});
