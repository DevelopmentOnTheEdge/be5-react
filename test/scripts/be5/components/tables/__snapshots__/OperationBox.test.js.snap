// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div
  className="operationList"
>
  <button
    className="btn btn-secondary btn-secondary-old btn-sm"
    disabled={true}
    onClick={[Function]}
  >
    Operation_1
  </button>
   
  <button
    className="btn btn-secondary btn-secondary-old btn-sm"
    disabled={true}
    onClick={[Function]}
  >
    Operation_2
  </button>
   
  <button
    className="btn btn-secondary btn-secondary-old btn-sm"
    disabled={true}
    onClick={[Function]}
  >
    Operation_3
  </button>
   
  <button
    className="btn btn-secondary btn-secondary-old btn-sm"
    disabled={true}
    onClick={[Function]}
  >
    Operation_4
  </button>
   
  <button
    className="btn btn-secondary btn-secondary-old btn-sm"
    disabled={true}
    onClick={[Function]}
  >
    Operation_5
  </button>
   
  <button
    className="btn btn-secondary btn-secondary-old btn-sm"
    disabled={true}
    onClick={[Function]}
  >
    Operation_6
  </button>
   
  <button
    className="btn btn-secondary btn-secondary-old btn-sm"
    disabled={true}
    onClick={[Function]}
  >
    Operation_7
  </button>
   
  <button
    className="btn btn-secondary btn-secondary-old btn-sm"
    disabled={true}
    onClick={[Function]}
  >
    Operation_8
  </button>
</div>
`;
